doc/examples/segmentation/plot_rolling_ball.py

```diff
@@ -0,0 +1,150 @@
+"""
+=======================
+Background Subtraction using the Rolling Ball Method
+=======================
+
+The rolling ball filter is a segmentation method that aims to separate the
+background from a grayscale image in case of uneven exposure. It is frequently
+used in biomedical image processing and was first proposed by Stanley R.
+Sternberg (1983) in the paper
+[Biomedical Image Processing](https://ieeexplore.ieee.org/document/1654163).
+
+The idea of the algorithm is quite intuitive. We think of the image as a
+surface that has unit sized blockes stacked on top of each other for each
+pixel. The number of blocks is determined by the intensity of a pixel. To get
+the intensity of the background at a desired position, we imagine submerging a
+ball into the blocks at the desired pixel position. Once it is completely
+covered by the blocks, the height of the ball determines the intensity of the
+background at that position. We can then *roll* this ball around below the
+surface to get the the background values for the entire image. As Sternberg
+puts it: "We can visualize a solid sphere that moves freely within the solid
+volume of the gel image umbra but is constrained by the upper surface of the
+umbra."
+
+To implement this algorithm in skimage we can use the following:
+"""
+
+from skimage.morphology import ball, erosion, dilation
+import numpy as np
```

Comment:
Please swap import order.
Reference: https://www.python.org/dev/peps/pep-0008/#imports

Reply:
fixed

Reply:
You want to import matplotlib before scikit-image. And you want to import it before numpy because of alphabetical order. See https://scikit-image.org/docs/dev/auto_examples/segmentation/plot_multiotsu.html for example.

Reply:
> And you want to import it before numpy because of alphabetical order

I don't think we enforce this at all? I generally import numpy before mpl every time.

Reply:
> I don't think we enforce this at all? I generally import numpy before mpl every time.

@jni sorry, I should have written: "And you might as well reorder imports alphabetically within import groups (since at this point you're reordering imports)" to mean suggestion rather than enforcement. That was indeed the intent, that's why I didn't comment any further when @FirefoxMetzger resolved the conversation without (back then, at that point) having moved the matplotlib import before the numpy import ;-)